doc/install_mne_python.rst

```diff
@@ -73,5 +73,17 @@ Install Python and MNE-Python
 
   If you get a new prompt with no error messages, you should be good to go!
 
+  .. note::
+
+     **Windows users:** If Mayavi plotting in Jupyter Notebooks doesn't work  
+     well, using the IPython magic `%gui qt` after importing  
```

Comment:
double-backticks on code, see how it just italicizes (as broken link) here:

https://6906-1301584-gh.circle-artifacts.com/0/tmp/circle-artifacts.NjLjWfq/html/install_mne_python.html